lib/view/list_settings_dialog.dart

```diff
@@ -81,12 +84,22 @@ class _ListSettingsDialogState extends State<ListSettingsDialog> {
         ],
       ),
       actions: [
+        if (widget.onDelete != null)
+          ElevatedButton.icon(
```

Comment:
You don't want two elevated buttons here. Elevated buttons are meant to draw the user's attention to a primary action. Deleting the list isn't the primary action in this context. The submit button is the primary button.

Reply:
Here is a good summary of buttons: https://m3.material.io/components/all-buttons

Reply:
> Elevated buttons are essentially filled tonal buttons with a shadow. To prevent shadow creep, only use them when absolutely necessary, such as when the button requires visual separation from a patterned background.

Really you could consider making the submit button a [`FilledButton`](https://api.flutter.dev/flutter/material/FilledButton-class.html)

Reply:
> > Elevated buttons are essentially filled tonal buttons with a shadow. To prevent shadow creep, only use them when absolutely necessary, such as when the button requires visual separation from a patterned background.
> 
> Really you could consider making the submit button a [`FilledButton`](https://api.flutter.dev/flutter/material/FilledButton-class.html)

Do you think I should do the same for the `EditItemDialog` submit button? Should extract them into a single `SubmitButton` class?

Reply:
Yeah, you should have consistent button style across your dialogs. It wouldn't hurt to have a `SubmitButton` class to help keep that consistent style.

Reply:
> Yeah, you should have consistent button style across your dialogs. It wouldn't hurt to have a SubmitButton class to help keep that consistent style.

Sounds good! Should I remove the modal sheet in the `ListWidget`? Or is that too much for this issue?

I think I might do that in issue #40 anyway, "Reorder Items", because in flutter's [`ReorderableListView`](https://api.flutter.dev/flutter/material/ReorderableListView-class.html#material.ReorderableListView.1), on mobile, items are made draggable/reorderable after long pressing on them, which is the same gesture we are using right now to pull up the modal bottom sheet.

Reply:
That is too much for this issue